addons/mail/static/src/components/channel_member/channel_member.scss

```diff
@@ -1,3 +1,7 @@
+.o_ChannelMember:hover {
+    background-color: gray('300');
+}
```

Comment:
Shouldn't this style only apply if hasOpenChat is true?

Reply:
I guess we should have an hover effect on guest members as well because it will look good. And guests are not having any pointer effect.